design/004-IoT-Adoption-Watcher.md

```diff
@@ -308,20 +319,90 @@ discard the older ones.
    Relevant parent ledger channel is the ledger channel between this user and
    the common intermediary.
 
+### Illustrative usage of watcher API
+
+The main component can either initialize a local watcher or connect to a remote
+watcher.
+
+#### Using the watcher from the main component
+
+```go
+// Either initialize a local watcher
+w, err := NewWatcher(rs RegistererSubscriber)
+
+// Or connect to the remote watcher
+w, err := ConnectToWatcherGrpc(cfg Config)
```

Comment:
`local` and `remote` can be two different packages. We can load the RegistererSubscriber in the watcher implementation.
```suggestion
// Either initialize a local watcher
w, err := local.NewWatcher(cfg local.Config)

// Or connect to the remote watcher
w, err := remote.NewWatcher(cfg remote.Config)
```

Reply:
Updated.